src/Microsoft.ML.Ensemble/OutputCombiners/Average.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace Microsoft.ML.Trainers.Ensemble
 {
-    public sealed class Average : BaseAverager, IRegressionOutputCombiner
+    internal sealed class Average : BaseAverager, IRegressionOutputCombiner
```

Comment:
Do we have any tests/ examples of how use Ensembles? I'm just curious how user will set any of combiners if they all hidden

Reply:
The current API for ensembles is not the one we want to make public, so I am internalizing it until we get a chance to develop a good public API for it. 

---
In reply to: [256207741](https://github.com/dotnet/machinelearning/pull/2505#discussion_r256207741) [](ancestors = 256207741)

Reply:
So if everything in this Ensemble is hidden, shall we exclude it from Microsoft.ML package? See Microsoft.ML.Ensemble.csproj  <IncludeInPackage>Microsoft.ML</IncludeInPackage>

---
In reply to: [256514419](https://github.com/dotnet/machinelearning/pull/2505#discussion_r256514419) [](ancestors = 256514419,256207741)

Reply:
Hmm. Good question. If there are any consumers of the nuget that use maml or entry points then we should probably leave it. 

---
In reply to: [256542187](https://github.com/dotnet/machinelearning/pull/2505#discussion_r256542187) [](ancestors = 256542187,256514419,256207741)